pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -547,6 +549,16 @@ private void initialize() {
       throw new RuntimeException("While creating out dir " + outDir.getPath(), e);
     }
 
+    initializeMqtt();
+  }
+
+  private void disconnectMqtt() {
+    Preconditions.checkState(mqttPublisher != null, "mqttPublisher not defined");
+    mqttPublisher.close();
```

Comment:
Does this handle any kind of completion of sending messages (whether that's desired or not), or just drop the connection?